uber/static/theme/prereg.css

```diff
@@ -1,3 +1,7 @@
+html, body {
+    min-height: 100%;
```

Comment:
This is an unrelated fix for the new preregistration theme for very short forms.